### PR TITLE
#5 new atmosphere model

### LINKED ISSUE #5: Simulation: Create an Atmospheric Model
The model will be used to calculate static pressure, static temperature, and density of air based on altitude.

We can use a standard model from NASA or other sources, please site the source in a comment.

### PR DESCRIPTION
**Pull request by Aarush26**
Created a function that calculates the temperature, pressure, and density of the atmosphere based on the input of a given altitude to address issue #5. The atmospheric model function is based off the NASA model [here](https://www.grc.nasa.gov/WWW/K-12/airplane/atmosmet.html). We are only concerned with the troposphere equations and constants, since we probably will not fly above 11,000 meters (we can dream though).

### CODE REVIEW

File: Simulation/AtmosphereModel.m [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+% broken up into three sections: troposphere, lower stratosphere, and upper stratosphere.
+% For the reality of this RC plane, we will only work on the model regarding the
+% troposphere constants. In the troposphere, temperature decreases linearly, while 
+% pressure decreases exponentially, and density is based off the values of 
+% temperature and pressure.
+
+% Atmosphere model is sourced from NASA at this link:
+% https://www.grc.nasa.gov/WWW/K-12/airplane/atmosmet.html
+
+% Create input altitude
```
Comment (TheGreatFez) [inline Simulation/AtmosphereModel.m]: Let's remove this comment since we changed the code to a function

**Review by TheGreatFez — CHANGES_REQUESTED** [review]
Just one small change

**Review by Aarush26 — COMMENTED** [review]
Deleted the comment that was above the function code because it referred to part of the code that was changed to make a function that used altitude as an input.

**Review by TheGreatFez — APPROVED** [review]
Looks great!

**Pull request merged**